app/services/oregon_digital/odd_endpoint_service.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module OregonDigital
+  class OddEndpointService
```

Comment:
Maybe rename to NonLDEndpointService or NonJsonLDEndpointService or something similar